backend/epub/minizip/unzip.c

```diff
@@ -600,9 +600,6 @@ local unzFile unzOpenInternal (const void *path,
 
     int err=UNZ_OK;
 
-    if (unz_copyright[0]!=' ')
```

Comment:
You can also remove `unz_copyright` vble.
https://github.com/mate-desktop/atril/blob/4852c2a1ec70597be303de82d01df68a54e3a44d/backend/epub/minizip/unzip.c#L122-L123

Reply:
thanks, done